app/_locales/pl/messages.json

```diff
@@ -127,6 +127,9 @@
   "cancel": {
     "message": "Anuluj"
   },
+  "cancelEdit": {
+    "message": "Cancelar edição"
```

Comment:
This is a portuguese translation... pretty sure it shouldn't be on the polish translations file